spk/links/Makefile

```diff
@@ -0,0 +1,21 @@
+SPK_NAME = links
+SPK_VERS = 2.12
+SPK_REV = 1
+SPK_ICON = src/links.png
+
+DEPENDS = cross/links
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Links is a web browser running in text mode.
+DISPLAY_NAME = Links
+BETA = 1
+
+HOMEPAGE   = http://links.twibright.com/
```

Comment:
A suggestion for your next PR: I don't mind the aligning, but let's use the minimum number of spaces to achieve that ;)